kubernetes/multicloud-pipeline/blueprint.yaml

```diff
@@ -0,0 +1,98 @@
+apiVersion: xl/v2
+kind: Blueprint
+
+metadata:
```

Comment:
Maybe we should just merge this with app-with-provider as both them seems to do almost the same thing